data/sql_updates/create_committee_history.sql

```diff
@@ -79,35 +67,33 @@ select distinct on (dcp.cmte_sk, cycle)
     dcp.cmte_custodian_title as custodian_name_title,
     dcp.cmte_custodian_zip as custodian_zip,
     -- Properties
-    dcp.cmte_email as email,
+    fec_yr.cmte_email as email,
     dcp.cmte_fax as fax,
-    dcp.cmte_web_url as website,
+    fec_yr.cmte_url as website,
     dcp.form_tp as form_type,
     dcp.leadership_pac as leadership_pac,
-    dcp.load_date as load_date,
```

Comment:
The `load_date` column comes from `dimcmteproperties`, which we're doing our best to stop using. It also looks like an field that captures internal FEC procedures (when a record was entered into the database) rather than information from the forms (when a report was filed). We can use `date_entered` and `date_changed`, but I'm not sure these fields add value.